### DISCUSSION TITLE
Merging Snap Targets

### DISCUSSION THREAD

**Discussion by Jonas-Lieske** [rxhanson/RectanglePro-Community#440] [Ideas]
What I'm missing from Powertoys is the functionality to merge snap targets when hovering a window between them. Meaning when I have a snap target for each half of the screen and I move the window between them, it will use the space of both areas, so the full screen.
This would make the creation of custom layouts so much better.

**Comment by rxhanson** [comment]
Thanks for the feedback! This one is on my backlog, but I don't have an ETA for it at this time.
